### PR TITLE
Update Pal.qml for new Ban button

### PR DESCRIPTION
**Pull request by digisomni**
Fix Pal ban buttons overlapping with width (maybe...) and add descriptions for new ban button.

### CODE REVIEW

File: interface/resources/qml/hifi/Pal.qml [hunk 1]
```diff
@@ -755,7 +755,8 @@ Rectangle {
                 onClicked: letterbox(hifi.glyphs.question,
                                      "Admin Actions",
                                      "<b>Silence</b> mutes a user's microphone. Silenced users can unmute themselves by clicking &quot;UNMUTE&quot; on their toolbar.<br><br>" +
-                                     "<b>Ban</b> removes a user from this domain and prevents them from returning. Admins can un-ban users from the Sandbox Domain Settings page.");
+                                     "<b>Ban (left)</b> identifies a user by username (if applicable) and machine fingerprint, then removes them from this domain and prevents them from returning. Admins can un-ban users from the Server Domain Settings page.<br><br>" +
+                                     "<b>Hard Ban (right)</b> identifies a user by username (if applicable), machine fingerprint, and IP address, then removes them from this domain and prevents them from returning. Admins can un-ban users from the Sandbox Domain Settings page.");
```
Comment (ctrlaltdavid) [inline interface/resources/qml/hifi/Pal.qml:759]:
```suggestion
                                     "<b>Hard Ban (right)</b> identifies a user by username (if applicable), machine fingerprint, and IP address, then removes them from this domain and prevents them from returning. Admins can un-ban users from the Server Domain Settings page.");
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-PR1155-e903c56-_d00711fa.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-PR1155-e903c56-.tar_d00711fa.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-PR1155-e903c56-.tar_d00711fa.Z

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-Quest-Alpha-PR1155-e903c56_d00711fa.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-PR1155-e903c56_d00711fa.apk

**Review by ctrlaltdavid — APPROVED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-PR1155-1c8026c-.tar_8c929108.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-PR1155-1c8026c-_8c929108.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-PR1155-1c8026c-.tar_8c929108.Z

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-PR1155-1c8026c-_8c929108.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-PR1155-1c8026c-_8c929108.exe

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-PR1155-1c8026c_8c929108.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1155/Vircadia-Quest-Alpha-PR1155-1c8026c_8c929108.apk

**Comment by digisomni** [comment]
image

image

That last one was before the CR.

**Pull request merged**